packages/core/src/Modules/Cashier/Containers/p2p-cashier.jsx

```diff
@@ -43,10 +43,11 @@ class P2PCashier extends React.Component {
         }
     }
     render() {
-        const { currency, local_currency_config, is_virtual, residence, setNotificationCount } = this.props;
+        const { currency, local_currency_config, is_virtual, residence, setNotificationCount, location } = this.props;
         const { order_id } = this.state;
         return (
             <P2P
+                {...(/verification/.test(location.hash) && { show_verification: true })}
```

Comment:
- Since this is a boolean, please use `is`, `should`, or `has` prefixes.
- For readability you could write this as:

```suggestion
                should_show_verification={/verification/.test(location.hash) ? true : undefined}
```

Reply:
done